Test.C

```diff
@@ -183,8 +183,44 @@ int main(){
         //selling equipment
         //decrease amount in index amount if things are sold
         //increase credit if things are sold
+        //So this is a lot of pseudo code or not finished but idk how to do it fully but it a good framework 
+        //assumes that you can sell the equipment for the same price that is bought 
+        int equipSellSelect;
+        int equipSellAmount;
+        int equipSellTransaction;
+        int equipSellAnswer;
+        
+        printf("|----------------------------- Equipment Selling ------------------------------|\n");
+        printf("Equipment that you own:\n");
+        //print equipment that is owned 
+        printf("Your current credits are: %d\n", player.currentCredits); //current credit
+        printf("Please select the equipment you would like to sell or select 11 to pass: "); //chose 11 becuase there are 10 options rn
```

Comment:
Good comment, but this is still unnecessary hard coding. You could make this dynamically update with our code by using the constant `NUM_EQUIPMENT`.
```suggestion
        printf("Please select the equipment you would like to sell or select %d to pass: ", NUM_EQUIPMENT + 1); //chose 11 becuase there are 10 options rn
```